evap/staff/tests/test_views.py

```diff
@@ -467,12 +459,14 @@ def test_view_list_sorting(self):
         self.assertGreater(position_evaluation1, position_evaluation2)
         self.app.reset()  # language is only loaded on login, so we're forcing a re-login here
 
-        self.manager.language = 'de'
-        self.manager.save()
-        page = self.app.get(self.url, user=self.manager).body.decode("utf-8")
-        position_evaluation1 = page.find("Evaluation 1")
-        position_evaluation2 = page.find("Evaluation 2")
-        self.assertLess(position_evaluation1, position_evaluation2)
+        # Re-enter staff mode, since the session was just reset
+        with run_in_staff_mode(self):
```

Comment:
here too :)

Reply:
This one is necessary, because the app (and therefore the session) is reset in the middle of the test. I added a comment in the code to explain it.

Reply:
oh, right. sorry.